R/NielsonKuparuk.R

```diff
@@ -0,0 +1,278 @@
+##Neilson, 2017
+##Megan Nguyen
+##2018
+
+#Clean files
+IC_WeirQ <- read.csv("/home/mnguyen/Neilson/IC_WeirQ.csv", header = TRUE)
+IC_WeirQ <- IC_WeirQ[,-3]
+names(IC_WeirQ) <- c("Date_adt", "Discharge_cms")
+
+Kup2AQ <- read.csv("/home/mnguyen/Neilson/Kup2AQ.csv", header = TRUE)
+names(Kup2AQ)
+
+ICKUP_Q <- read.csv("/home/mnguyen/Neilson/ICKUP_Q.csv", header = TRUE)
+str(ICKUP_Q)
+results <- grepl(pattern = "^\\d", ICKUP_Q$Date_adt)
+delete <- c()
+for(i in 1:length(ICKUP_Q$Date_adt)){
```

Comment:
I do this all the time, too, but it's better practice to use `seq_along` in the place of `1:length()` because it fails more gracefully.  For example, if the `length` is zero, then you have your loop going from 1 to 0 (1:0).  

https://stackoverflow.com/questions/24917228/proper-way-to-loop-over-the-length-of-a-dataframe-in-r